commit 6a769d9776687839537739bda72656e282c676c4 (refs/changes/07/2707/2)
Author: Sam Gwydir <sam.gwydir@joyent.com>
Date:   2017-10-06T21:09:38+00:00 (2 years ago)
    
    MANATEE-373 Want additional postgres modules for admin purposes
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/Makefile.postgres b/Makefile.postgres
index cdd2ecd..c02e935 100644
--- a/Makefile.postgres
+++ b/Makefile.postgres
@@ -82,9 +82,36 @@ pg96:
 	cd $(DEPSDIR)/postgresql96 && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/auto_explain && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/oid2name && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pageinspect && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_buffercache && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_freespacemap && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_prewarm && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/postgresql96/contrib/pg_stat_statements && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_visibility && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pgstattuple && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/vacuumlo && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/pg_repack && env \
 		CFLAGS=-m64 \
 		PATH=$(RELSTAGEDIR)/root/$(PG96DIR)/bin:$$PATH \
